mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/services/SanityService.kt

```diff
@@ -24,6 +27,7 @@ import kotlin.collections.set
 class SanityService(
```

Comment:
Side-note relatert til noe annet vi snakket om tidligere i uken - det gir kanskje mening å endre navn på denne til f.eks. `SanityTiltakService` for å tydeliggjøre hvilke data denne tjenesten jobber med. Kan evt. gjøres i egen PR om du er enig :+1:

Reply:
Tar det i egen PR og prøver å ikke endre på `SanityService` nå i denne pren